root_profiler/sheets/standard/v3.5-DD.php

```diff
@@ -845,6 +845,42 @@ function GetStaticHelp( $name, $staticHelp, $class = "help" )
             </tr>
           </table> <!-- skillsandgear -->
 
+          <input type="checkbox" <?php getnc('ContainerDisp'); ?> onchange="ToggleDisplay('containers', this);" style="width:15px; border:none;"/>
```

Comment:
> It would be cleaner to put the style in the .css rather than an attribute here.

I just followed the coding convention of all the other feature-enabling checkboxes. If you fix this, you should fix all the others in root_profiler/sheets/standard/v3.5-DD.php

Reply:
That's fine, just noting what would be cleaner.  We can worry about that another day.